scene/resources/surface_tool.cpp

```diff
@@ -1013,6 +1024,24 @@ void SurfaceTool::generate_normals(bool p_flip) {
 	}
 }
 
+void SurfaceTool::_apply_smoothing_group(HashMap<Vertex, Vector3, VertexHasher> &r_vertex_hash, unsigned int p_from, unsigned int p_to, bool &r_smooth) {
```

Comment:
```suggestion
void SurfaceTool::_apply_smoothing_group(HashMap<Vertex, Vector3, VertexHasher> &r_vertex_hash, uint32_t p_from, uint32_t p_to, bool &r_smooth) {
```

Just a couple more unsigned int renames